client/state/sites/test/selectors.js

```diff
@@ -3921,4 +3922,35 @@ describe( 'selectors', () => {
 			expect( noNewAttributes.isSiteUpgradeable ).to.have.property;
 		} );
 	} );
+
+	describe( '#getUpdatesBySiteId()', () => {
```

Comment:
general question: do you think selector tests like this are useful?

I struggle to see the value. What kind of situation would this catch a bug? The most obvious thing I see breaking is state shape changing and the selector not working anymore -- but these tests would never catch that since shape is provided by the test.